### ISSUE TITLE
When sending a collectible, txn history as # for amount value

### ISSUE THREAD

**Issue by ibrahimtaveras00** [MetaMask/metamask-mobile#1563]
**Describe the bug**

Send a collectible and look at the txn history view.  There is a # in the Amount value

**Screenshots**

![image](https://user-images.githubusercontent.com/41752922/81751838-3be6e800-947e-11ea-97ae-3dac6a2b66cf.png)

**To Reproduce**
_Steps to reproduce the behavior_

Send a collectible then look at the txn history for that txn

**Expected behavior**

@cjeria maybe we can put something in this place or leave blank 

**Smartphone (please complete the following information):**
 - Device: all
 - OS: any
 - Version all versions 

-------------------------------------------------------------
_to be added after bug submission by internal support / PM_
**Severity**

- How critical is the impact of this bug on a user?
- Add stats if available on % of customers impacted
- Is this visible to all users?
- Is this tech debt?

**Comment by estebanmino** [comment]
I don't think this is a bug. 32 is the id of the collectible right? Which collectible is it? It seems like you don't have that kind of collectible anymore so the app doesn't know the name, therefore you get a `Send Collectible` title instead of `Send ${name of collectible}`

**Comment by ibrahimtaveras00** [comment]
Here's the flow from current implementation in prod (taken from v02.15.(405) on iOS as an example)

Account 1 has crypto kitty and sends to Account 2 = http://recordit.co/qCIWVbRfXN

This is the view how it looks right away as soon as it just got sent:
![Screen Shot 2020-05-13 at 5 19 31 PM](https://user-images.githubusercontent.com/41752922/81866982-01dd1b00-953e-11ea-8413-e7bbe230c6c6.png)


This is the view how it looks when I switch to account 2 (which now owns it) and then switch back to account 1:

![Screen Shot 2020-05-13 at 5 20 47 PM](https://user-images.githubusercontent.com/41752922/81867054-1ae5cc00-953e-11ea-92d9-b87272c6a4c4.png)

As you can see, similar to above, this isn't something newly introduced and has been around.  One thing to take note of is the behavior of how Sent Collectible acts.  After sending the collectible and coming back to the account you no longer own the collectible thus cryptokitty disappeared and just shows the ID #.  This happens to pretty much all the types of collectibles I own.

**Comment by ibrahimtaveras00** [comment]
CC: @cjeria

**Comment by ibrahimtaveras00** [comment]
You're correct @estebanmino in my first screenshot that is the ID of the collectible

Similar to my above 2 screenshots which is also the ID.  I guess the question here is, should we be showing the ID# as the Amount and in the total?

**Comment by estebanmino** [comment]
I think this is not an issue anymore 
![image](https://user-images.githubusercontent.com/12115171/116164901-a7928700-a6c8-11eb-98ac-84e30715cfb9.png)

**Comment by mobularay** [comment]
@ibrahimtaveras00 to check in if this is still an issue... a quick sync b/n Omna Ibrahim Christian

**Comment by mobularay** [comment]
Discussion b/n @jakehaugen, Tian and @cjeria : has not been actively worked on since May 10 sprint. Removing from sprint countdown until this work is going to be picked up again.
